tests/integration/test_vulnerability_detector/test_providers/data/test_cases/test_multiple_feeds_provider.yaml

```diff
@@ -0,0 +1,24 @@
+- name: 'RedHat 8'
+  description: 'Insert RHEL 8 OVAL and JSON feed from local path'
+  configuration_parameters:
+    PROVIDER: 'redhat'
+    OS: '8'
+    OS_PATH: CUSTOM_REDHAT_OVAL_FEED_PATH
+    PATH: CUSTOM_REDHAT_JSON_FEED_PATH
+  metadata:
+    provider_name: 'Red Hat Enterprise Linux 8'
+    provider_json_name: 'JSON Red Hat Enterprise Linux'
+    oval_feed_path: CUSTOM_REDHAT_OVAL_FEED_PATH
+    json_feed_path: CUSTOM_REDHAT_JSON_FEED_PATH
+
+- name: 'Debian Buster'
+  description: 'Insert Debian Buster OVAL and JSON feed from local path'
+  configuration_parameters:
+    PROVIDER: 'debian'
+    OS: 'Buster'
+    OS_PATH: CUSTOM_DEBIAN_OVAL_FEED_PATH
+    PATH: CUSTOM_DEBIAN_JSON_FEED_PATH
+  metadata:
+    provider_name: 'Debian Buster'
+    oval_feed_path: CUSTOM_DEBIAN_OVAL_FEED_PATH
+    json_feed_path: CUSTOM_DEBIAN_JSON_FEED_PATH
```

Comment:
adds new line at the end

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2538/commits/762b15217466b07a5e4bfa23bcf2c647e1848760